### PR TITLE
rpc: unify RPC handlers and metrics

### PR DESCRIPTION
**Pull request by fyrchik**
This will make it easier to enable/disable RPC when needed (e.g. `getnep5*`) and is easier to read.

This is a draft for a discussion.

### CODE REVIEW

File: pkg/rpc/server/prometheus.go [hunk 1]
```diff
@@ -1,249 +1,63 @@
...
+		"getclaimable",
+		"getconnectioncount",
+		"getcontractstate",
+		"getvalidators",
+		"getnep5balances",
+		"getnep5transfers",
+		"getversion",
+		"getpeers",
+		"getrawmempool",
+		"validateaddress",
```
Comment (roman-khimov) [inline pkg/rpc/server/prometheus.go]: Not a sorted list. And I think adding `invoke*` there is also useful, every call should have a metric associated.

Comment (fyrchik) [reply pkg/rpc/server/prometheus.go]: Done!

File: pkg/rpc/server/prometheus.go [hunk 2]
```diff
@@ -1,249 +1,66 @@
...
-		getunclaimedCalled,
-		getunspentsCalled,
-		gettransactionheightCalled,
-		gettxoutCalled,
-		getrawtransactionCalled,
-		sendrawtransactionCalled,
-		submitblockCalled,
-		getstorageCalled,
-	)
+	for i := range rpcCalls {
```
Comment (roman-khimov) [inline pkg/rpc/server/prometheus.go]: `for i := range rpcHandlers`?

Comment (fyrchik) [reply pkg/rpc/server/prometheus.go]: Yes, it makes sense now.

**Pull request merged**